server/middleware/index.js

```diff
@@ -1,6 +1,28 @@
 var habitat = require('habitat');
 var middleware = {};
 
+// For Webmaker Auth
+middleware.authenticateWithCookie = function (req, res, next) {
+    var user = req.session && req.session.user;
+
+    // Check auth
+    if (habitat.get('FAKE_AUTH')) {
+        user = req.body.user;
+    } else {
```

Comment:
This else clause shouldn't exist right? Imagine a case where fake auth is set, but I forget to send a user object.